index.html

```diff
@@ -4,38 +4,70 @@
+    />
+    <title>Quiz App by Carsten</title>
   </head>
   <body>
-    <h1 class="title">Quiz App</h1>
-    <h2 class="title">by Carsten Jahnke</h2>
-    <p class="question-card-title">Question Card 1</p>
-        <p class="question-card-1-box">
-        Lorem ipsum dolor, sit amet consectetur adipisicing elit. Autem animi
-        ratione dolorem non cum culpa necessitatibus recusandae aut! In voluptatem
-        ex culpa non enim omnis tempore dolor consectetur eos nulla?
-        <p class="question-card-1-answer">Question Card 1 Answer</p>
-        <p class="question-card-1-hashtagbox">Question Card 2 Hastagbox</p>
-    </p>
-    <p class="question-card-title">Question Card 2</p>
-    <p class="question-card-2-box">
-        Lorem ipsum dolor, sit amet consectetur adipisicing elit. Autem animi
-        ratione dolorem non cum culpa necessitatibus recusandae aut! In voluptatem
-        ex culpa non enim omnis tempore dolor consectetur eos nulla?
-        <p class="question-card-2-answer">Question Card 2 Answer</p>
-        <p class="question-card-2-hashtagbox">Question Card 2 Hashtagbox</p>
-        <footer>
-            <section class="footer">Question Card 1 Answer
-                <nav id="footer-nav">
-                    <ul id="footer-nav_list">
-                        <li><a class="footer-navigation" href="#home">Home</a></li>
-                        <li><a class="footer-navigation" href="#bookmark">Bookmark</a></li>
-                        <li><a class="footer-navigation" href="#profile">Profile</a></li>
-                    </ul>
-                </nav>
-            </section>
-        </footer>
-    </p>
+    <script type="text/javascript" src="js/headerscroll.js"></script>
+    <div id="header">Quiz App</div>
+    <section id="home">
+        <section class="question-boxes">
+        <h3 class="question-card-title">Question 01</h3>
+        <p class="questions">What property flips the axes in flexbox?</p>
+        <button class="button-answer">Show Answer</button>
+        <div class="category">
+            <p>#html</p>
+            <p>#css</p>
+            <p>#flexbox</p>
+        </div>
+        </section>
+        <section class="question-boxes">
+        <h3 class="question-card-title">Question 01</h3>
+        <p class="questions">What property flips the axes in flexbox?</p>
+        <button class="button-answer">Show Answer</button>
+        <div class="category">
+            <p>#html</p>
+            <p>#css</p>
+            <p>#flexbox</p>
+        </div>
+        </section>
+        <section class="question-boxes">
+        <h3 class="question-card-title">Question 01</h3>
+        <p class="questions">What property flips the axes in flexbox?</p>
+        <button class="button-answer">Show Answer</button>
+        <div class="category">
+            <p>#html</p>
+            <p>#css</p>
+            <p>#flexbox</p>
+        </div>
+        </section>
+    </section>
```

Comment:
Biased note: The code indentation here made my brain spiral a few times 😵‍💫 it could use more tree like indentation.